packages/contracts-periphery/scripts/deploy-nft-bridge.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
This script just works:

- run `make up` in `ops`
- run `./scripts/deploy-nft-bridge.sh`

Would like to sub out the networks and use this for the actual deployments

Reply:
You can use tags in the deployment scripts to make this simpler. I would add `L2ERC721BridgeDeployment` as a tag to the L2 contracts then `L1ERC721BridgeDeployment` as a tag to the L1 contracts. Then you can just do `npx hardhat deploy --tags L2ERC721BridgeDeployment` and it'll deploy everything with that tag, respecting dependencies.

Reply:
But honestly if this works, I don't care enough to make you edit this